ppafm/PPPlot.py

```diff
@@ -95,7 +95,7 @@ def plotImages(
         plt.imshow(F[i], origin="lower", interpolation=interpolation, cmap=cmap, extent=extent, vmin=vmin, vmax=vmax)
 
         if cbar:
-            plt.colorbar()
+            plt.colorbar(shrink=float(F[i].shape[0]) / float(F[i].shape[1]) if F[i].shape[0] < F[i].shape[1] else 1.0)
```

Comment:
I think this would be simpler (and maybe clearer) as:
```suggestion
            plt.colorbar(shrink=min(1.0, F[i].shape[0] / F[i].shape[1]))
```
No need for the `float()` in Python 3.